src/pages/home/HeaderView.js

```diff
@@ -39,51 +45,66 @@ const defaultProps = {
-                        styles.justifyContentBetween,
-                    ]}
-                >
+const HeaderView = (props) => {
+    const participants = lodashGet(props.report, 'participants', []);
+    const reportOption = {
+        text: lodashGet(props.report, 'reportName', ''),
+        tooltipText: getReportParticipantsTitle(participants),
+        participantsList: getPersonalDetailsForLogins(participants, props.personalDetails),
+    };
+
+    return (
+        <View style={[styles.appContentHeader]} nativeID="drag-area">
+            <View style={[styles.appContentHeaderTitle, !props.isSmallScreenWidth && styles.pl5]}>
+                {props.isSmallScreenWidth && (
                     <Pressable
-                        onPress={() => {
-                            const {participants} = props.report;
-                            if (participants.length === 1) {
-                                Navigation.navigate(ROUTES.getProfileRoute(participants[0]));
-                            }
-                        }}
+                        onPress={props.onNavigationMenuButtonClicked}
+                        style={[styles.LHNToggle]}
                     >
-                        <MultipleAvatars avatarImageURLs={props.report.icons} />
+                        <Icon src={BackArrow} />
                     </Pressable>
-                    <Header title={props.report.reportName} />
-                    <View style={[styles.reportOptions, styles.flexRow]}>
+                )}
+                {props.report && props.report.reportName ? (
+                    <View
+                        style={[
+                            styles.flex1,
+                            styles.flexRow,
+                            styles.alignItemsCenter,
+                            styles.justifyContentBetween,
+                        ]}
+                    >
                         <Pressable
-                            onPress={() => togglePinnedState(props.report)}
-                            style={[styles.touchableButtonImage, styles.mr0]}
+                            onPress={() => {
+                                if (participants.length === 1) {
+                                    Navigation.navigate(ROUTES.getProfileRoute(participants[0]));
+                                }
+                            }}
                         >
-                            <Icon src={Pin} fill={props.report.isPinned ? themeColors.heading : themeColors.icon} />
+                            <MultipleAvatars avatarImageURLs={props.report.icons} />
                         </Pressable>
+                        <View style={[styles.flex1, styles.flexRow]}>
+                            <OptionRowTitle
+                                option={reportOption}
+                                tooltipEnabled
+                                numberOfLines={2}
+                                tooltipContainerStyle={styles.dInline}
```

Comment:
It seems like `tooltipContainerStyle` can only ever contain `styles.dInline` and that is already the default property for `tooltipContainerStyle`. I think I would suggest removing the `tooltipContainerStyle` prop entirely.